roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -976,13 +979,55 @@ RooAbsReal* RooAbsPdf::createNLL(RooAbsData& data, const RooLinkedList& cmdList)
   pc.defineDouble("IntegrateBins", "IntegrateBins", 0, -1.);
   pc.defineMutex("Range","RangeWithName") ;
   pc.defineMutex("GlobalObservables","GlobalObservablesTag") ;
+  pc.defineInt("NewStyle", "NewStyle", 0, 0);
+  pc.defineMutex(
+    "NewStyle",
+    "NumCPU"); // New style likelihoods define parallelization through Parallel(...) on RooMinimizer or fitTo.
```

Comment:
The RooMinimizer has no `Parallel` option now, so this comment doesn't make sense I think